app/room_request/views.py

```diff
@@ -103,44 +103,49 @@ def delete_room_request(room_request_id):
     return redirect('/room-request/')
 
 
-@room_request.route('/<int:form_id>', methods=['GET', 'POST'])
-def viewID(form_id):
-    form = ActivityForm()
-    trasnferf = TransferForm()
-    user_found = False
-    comments = Activity.query.filter_by(room_request_id = form_id)
-    name = ""
-    try:
-        request_ob = RoomRequest.query.get(form_id)
-        print("User found")
-        name = str(request_ob.first_name) + " " + str(request_ob.last_name)
```

Comment:
Instead of passing in `name`, I decided to pass in the entire `room_request` object, since later we'll be using this same route to display all the room request's data.